R/nm-join.R

```diff
@@ -109,10 +115,10 @@ nm_join <- function(
     stop(glue("couldn't find `.join_col` {.join_col} in data with cols: {paste(names(.d), collapse = ', ')}"))
   }
 
-  if(.d[.join_col] %>% anyDuplicated() != 0)
+  if(anyDuplicated(.d[.join_col]) != 0)
   {
    dup_row <- .d[.join_col][duplicated( .d[.join_col]) %>% which(),]
-   stop(glue("Duplicate rows in data in {.join_col}: {dup_row}"))
+   stop(glue("Duplicate rows in data in {.join_col}: {dup_row %>% head()}"))
   }
```

Comment:
If we `head()` the duplicate rows, we might want to write `...truncated` if there are more duplicate values beyond the default of 6. My vote would be to remove the printing of the duplicate rows entirely, and just say:
```
stop(glue("Duplicate rows were found in {.join_col}. Please see `?nm_join` for more details"))
```